test/test-creation.js

```diff
@@ -973,8 +973,8 @@ describe('JHipster generator', () => {
                 `${TEST_DIR}features/user/user.feature`
             ]);
             assert.noFile([
-                `${TEST_DIR}gatling/gatling.conf`,
-                `${TEST_DIR}gatling/logback.xml`
+                `${TEST_DIR}gatling/conf/gatling.conf`,
```

Comment:
Apparently, Github do not show when we move files. But I correctly moved the two files in generator-jhipster/generators/server/templates/src/test/gatling/conf/.